keyboards/keebio/irispad/rev8/info.json

```
@@ -0,0 +1,151 @@
+            "pixel_flow": true,
+            "pixel_fractal": true,
+            "pixel_rain": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "solid_multisplash": true,
+            "solid_reactive": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_multinexus": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_simple": true,
+            "solid_reactive_wide": true,
+            "solid_splash": true,
+            "splash": true,
+            "typing_heatmap": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [0, 0], "x": 0, "y": 5, "flags": 4},
+            {"x": 18, "y": 5, "flags": 2},
+            {"matrix": [0, 1], "x": 37, "y": 5, "flags": 4},
+            {"matrix": [0, 2], "x": 76, "y": 2, "flags": 4},
+            {"x": 91, "y": 1, "flags": 2},
+            {"matrix": [0, 3], "x": 110, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 146, "y": 5, "flags": 4},
+            {"x": 165, "y": 3, "flags": 2},
+            {"matrix": [0, 5], "x": 183, "y": 3, "flags": 4},
+            {"matrix": [1, 5], "x": 183, "y": 17, "flags": 4},
+            {"matrix": [1, 4], "x": 137, "y": 15, "flags": 4},
+            {"matrix": [1, 3], "x": 110, "y": 13, "flags": 4},
+            {"matrix": [1, 2], "x": 73, "y": 15, "flags": 4},
+            {"matrix": [1, 1], "x": 37, "y": 18, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 18, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [2, 1], "x": 37, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 73, "y": 28, "flags": 4},
+            {"matrix": [2, 3], "x": 110, "y": 27, "flags": 4},
+            {"matrix": [2, 4], "x": 137, "y": 28, "flags": 4},
+            {"matrix": [2, 5], "x": 183, "y": 30, "flags": 4},
+            {"matrix": [3, 5], "x": 183, "y": 43, "flags": 4},
+            {"matrix": [3, 4], "x": 146, "y": 42, "flags": 4},
+            {"matrix": [3, 3], "x": 110, "y": 40, "flags": 4},
+            {"x": 91, "y": 50, "flags": 2},
+            {"matrix": [3, 2], "x": 73, "y": 42, "flags": 4},
+            {"matrix": [3, 1], "x": 37, "y": 45, "flags": 4},
+            {"x": 18, "y": 43, "flags": 2},
+            {"matrix": [3, 0], "x": 0, "y": 45, "flags": 4},
+            {"matrix": [4, 5], "x": 224, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 206, "y": 64, "flags": 4},
+            {"x": 183, "y": 58, "flags": 2},
+            {"matrix": [4, 3], "x": 165, "y": 58, "flags": 4},
+            {"matrix": [4, 2], "x": 128, "y": 47, "flags": 4}
+        ],
+        "max_brightness": 180,
+        "sleep": true
+    },
+    "usb": {
+        "device_version": "8.0.0",
```

Currently doesn't build.
```suggestion
        "device_version": "8.0.0",
        "vid": "0xCB10",
```

---

Strange, since I have it in keyboard-level info.json, and it builds on my machine, but I've committed the change here.

---

Ah, I forgot to commit that info.josn